modules/rosa-cluster-autoscaler-ui-settings.adoc

```diff
@@ -81,22 +108,22 @@ The tables explain all the configurable UI settings when using cluster autoscali
 |Type or Range
 |Default
 
-|`enable`
+|`enabled`
 |Should the cluster autoscaler scale down the cluster.
 |`boolean`
 |`true`
 
-|`scaleDown.unneededTime`
-|How long a node should be unneeded before it is eligible for scale down.
-|`string`
-|10m
-
-|`scaleDown.utilizationThreshold`
+|`scale-down-utilization-threshold`
 |Node utilization level, defined as sum of requested resources divided by capacity, below which a node can be considered for scale down.
 |`string`
```

Comment:
```suggestion
|`float`
```
it's a bit misleading/tricky the way we have it in the API, because it's a string field in the API but we don't anticipate anything other than a float as a value
the reason for that is that in the openshift schema it's also this way, so I didn't want to make something that will restrict us in the future in case they'll add another possible value that is not a float
so practically speaking it's a float, and we can have it this way in all the clients and in the docs. if for some reason we'll need to change it then we can probably change those
@celdrake I assume in the UI it's only restricting it to a float (which is what I would expect)

Reply:
@osherdp Based on the tests that we did, the current restriction in the UI is that it must be a float between 0 and 1. Is that correct?

Reply:
@celdrake yeah that's perfect